src/server/server_family.cc

```diff
@@ -554,6 +550,58 @@ string_view GetRedisMode() {
   return ClusterConfig::IsEnabledOrEmulated() ? "cluster"sv : "standalone"sv;
 }
 
+struct ReplicaOfArgs {
+  bool no_one = false;
+  string_view host;
+  string_view port_sv;
```

Comment:
I don't follow, sorry.
Why not have `uint16_t` and have it be `0` in case is it replica of no-one?
Then instead of `port_sv` member you could have a `std::string GetPortString()`, which will not allocate memory because of small string optimization (but in any case it's not important in this flow)

Reply:
Not that this is critical :) It's a nit.